Tests/Microsoft.Azure.Mobile.Test.Windows/StartServiceLogTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.Azure.Mobile.Ingestion.Models;
+using Microsoft.Azure.Mobile.Utils;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Mobile.Test.Windows
+{
+    [TestClass]
+    public class StartServiceLogTests
+    {
+        private const string StorageTestChannelName = "storageTestChannelName";
+
+        [TestInitialize]
+        public void asdf()
+        {
+            LogSerializer.AddFactory(StartServiceLog.JsonIdentifier, new LogFactory<StartServiceLog>());
+        }
+
+        [TestMethod]
+        public void CheckInitialValues()
+        {
+            StartServiceLog log = new StartServiceLog();
+            Assert.IsNull(log.Device);
+            Assert.IsNull(log.Services);
+            Assert.IsNull(log.Sid);
+            Assert.AreEqual(0, log.Toffset);
+        }
+
+        [TestMethod]
+        public void SaveStartServiceLog()
```

Comment:
Maybe this should be split into different test cases? It seems like it is testing a lot of different things in one method